src/DisplayBoard.cpp

```diff
@@ -30,14 +38,20 @@ std::string display(const Board& b)
     auto height = b.getSizeCols();
     auto widthRows = b.getLongestCluesLenghtInRows();
     auto heightCol = b.getLongestCluesLenghtInCols();
+    const Lines cluesCols = b.getCluesCols();
```

Comment:
you don't need variables if you use this value only once. In this case pass it directly into getFormattedColumns function.
string clues can also be skipped and getFormattedColumns function call can be passed below.